apis/validator/block.v2.yaml

```diff
@@ -18,8 +18,8 @@ get:
         $ref: "../../beacon-node-oapi.yaml#/components/schemas/Uint64"
     - name: randao_reveal
       in: query
-      required: true
-      description: "The validator's randao reveal value."
+      required: false
```

Comment:
If this is empty, what is BN supposed to return? 0x00? random?

would it be simpler to just keep this as *required* but that the caller can/should fill in the junk if `verify_randao=false`